src/translation.h

```diff
@@ -221,5 +221,6 @@ std::ostream &operator<<( std::ostream &out, const translation &t );
 std::string operator+( const translation &lhs, const std::string &rhs );
 std::string operator+( const std::string &lhs, const translation &rhs );
 std::string operator+( const translation &lhs, const translation &rhs );
+bool operator<( const translation &lhs, const translation &rhs );
```

Comment:
Not sure why the compilers were not complaining about this but it should be removed,